scripts/Diffraction/isis_powder/osiris_routines/osiris_param_mapping.py

```diff
@@ -25,5 +26,15 @@
     ParamMapEntry(ext_name="xye_filename", int_name="xye_filename"),
     ParamMapEntry(ext_name="sample_empty_scale", int_name="sample_empty_scale", optional=True),
     ParamMapEntry(ext_name="keep_raw_workspace", int_name="keep_raw_workspace", optional=True),
+    ParamMapEntry(ext_name="absorb_corrections", int_name="absorb_corrections"),
```

Comment:
I think this should be named in a way that is consistent with the other instruments e.g.
https://github.com/mantidproject/mantid/blob/5096764bb676ca2be248be1fdcb690e0fbd1990a/scripts/Diffraction/isis_powder/polaris_routines/polaris_param_mapping.py#L20

Reply:
I was going to do this but I noticed the previous parameters don't follow the convention of the other instruments (e.g. vanadium_normalization not do_vanadium_normalization). If I changed all of the parameters the users using the existing script will have to change it. In addition, the OSIRIS users probably don't use other instrument scripts so they will not notice this inconsistency. What do you think?